.jenkins/Jenkinsfile

```diff
@@ -82,7 +82,7 @@ def ACCContainerTest(String label, String version) {
                         ninja -v
                         ctest --output-on-failure
                        """
-                dcap.ContainerRun("${DOCKER_REGISTRY}/oetools-full-${version}:latest", "clang-8", task, "--cap-add=SYS_PTRACE --device /dev/sgx:/dev/sgx")
+                dcap.ContainerRun("${DOCKER_REGISTRY}/dcapdockerciregistry-ubuntu${version}:latest", "clang-10", task, "--cap-add=SYS_PTRACE --device /dev/sgx:/dev/sgx --volume /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket")
```

Comment:
What creates this Docker image? Also note the registry name is being duplicated in the name, so the full image name ends up being `dcapdockerciregistry.azurecr.io/dcapdockerciregistry-ubuntu${version}:latest`